docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/BinaryClassification/StochasticGradientDescentWithOptions.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Microsoft.ML;
+using Microsoft.ML.Trainers;
+
+namespace Microsoft.ML.Samples.Dynamic.Trainers.BinaryClassification
+{
+    public static class StochasticGradientDescentWithOptions
+    {
+        // In this examples we will use the adult income dataset. The goal is to predict
+        // if a person's income is above $50K or not, based on different pieces of information about that person.
```

Comment:
>r not, based on different pieces of information [](start = 47, length = 47)

Ditto from the other sample.

Reply:
changes all samples to use "demographic information" instead.

---
In reply to: [259451108](https://github.com/dotnet/machinelearning/pull/2688#discussion_r259451108) [](ancestors = 259451108)